CONTRIBUTING.rst

```diff
@@ -680,6 +680,18 @@ Pull Requests
   refactorings or minor unrelated changes. The title of merge commit in this
   case usually matches that of the main commit.
 
+* Github lets any user with write access resolve comments to changes in a PR. We
+  aren't that permissive. When the reviewer makes a comment, either requesting
+  a change or asking a question, the author addresses the comment by either
+
+  - making the requested changes and reacting to the comment with thumbs-up
+
+  - or answering the question in a reply to the comment.
+
+  In either case, only the reviewer resolves the comment. This is to ensure that
```

Comment:
This doesn't address the case where the reviewer's suggestions were incorrect or couldn't be applied without caveats.